app/services/invoices/compute_taxes_and_totals_service.rb

```diff
@@ -19,6 +19,12 @@ def call
         after_commit { Invoices::ProviderTaxes::PullTaxesAndApplyJob.perform_later(invoice:) }
 
         return result.unknown_tax_failure!(code: "tax_error", message: "unknown taxes")
+      elsif customer.vies_check_in_progress?
+        invoice.status = "pending" if finalizing
+        invoice.tax_status = "failed"
```

Comment:
That would enable action in frontend to re-sync tax failed invoice with tax provider (Invoices::RetryService) - not sure we want it for vies? Also, how would the invoice  switch from failed status in this case?